hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -440,6 +440,14 @@
       clock_reset_export: ["ast"],
       domain: "Aon",
       base_addr: "0x40440000"
+    },  
+    { name: "pwm_aon",
+      type: "pwm",
+      clock_srcs: {clk_i: "io_div4", clk_core_i: "aon"},
+      clock_group: "powerup",
+      reset_connections: {rst_ni: "sys_io_div4", rst_core_ni: "sys_aon"},
+      domain: "Aon",
+      base_addr: "0x40450000",
```

Comment:
I think you are missing the associated entry in the crossbar configuration (xbar_peri.hjson).

Reply:
Oh, my! Thanks for the feedback.   I think you are right.  I forgot that this time.  That may explain why the CI is failing.